ledger/acctupdates.go

```diff
@@ -90,17 +90,38 @@ func (au *accountUpdates) loadFromDisk(l ledgerForTracker) error {
 		return fmt.Errorf("accountUpdates.loadFromDisk: initAccounts not set")
 	}
 
+	latest := l.Latest()
 	err := au.dbs.wdb.Atomic(func(tx *sql.Tx) error {
 		var err0 error
-		err0 = accountsInit(tx, au.initAccounts, au.initProto)
-		if err0 != nil {
-			return err0
+		initAux := func() (basics.Round, error) {
```

Comment:
Can this be a separate function?  Would be easier to understand.

Reply:
Done